packages/devextreme/testing/testcafe/tests/scheduler/viewOffset/multiCellSelection.ts

```diff
@@ -8,88 +9,96 @@ fixture.disablePageReloads`Offset: Multi cell selection`
 
 const SCHEDULER_SELECTOR = '#container';
 
+// TODO Vinogradov: Change it to a general dispose widget helper function.
+const disposeScheduler = ClientFunction(() => {
```

Comment:
Let create hook or function **dispose**? Targeted selector have equal name in all tests and all components. Create enum with components(like dxScheduler, dxGrid) and pass to function

Reply:
I agree with you
But let's create this ```dispose``` helper in a separate task :)